source/_includes/installation/core.md

```diff
@@ -102,3 +113,37 @@ If this address doesn't work you may also try `http://localhost:8123` or `http:/
 When you run the `hass` command for the first time, it will download, install and cache the necessary libraries/dependencies. This procedure may take anywhere between 5 to 10 minutes. During that time, you may get "site cannot be reached" error when accessing the web interface. This will only happen for the first time, and subsequent restarts will be much faster.
 
 </div>
+  
+### Starting Home Assistant on boot
```

Comment:
We consider this out of scope for our documentation. This is an advanced installation method that covers setting up Home Assistant itself, not how to administer your system.

These section have been removed from our documentation as is available on the community guides section on the forums (as an editable wiki).